core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -348,7 +348,9 @@ class UserService(
         platform,
         protectedData,
         region = maybeRegion,
-        organization = Option(p.getOrganization).map(org => RawlsBillingProjectOrganization(org.isEnterprise))
+        organization = Option(p.getOrganization).map(org =>
+          RawlsBillingProjectOrganization(org.isEnterprise, org.getLimits.asScala.toMap)
```

Comment:
Since the imported `Organization` is java, there's a lot of compatibility issues between java map and scala map.  I have a lot of `asJava` and `asScala` scattered around and would love input on any better ways to deal with this.

Reply:
I don't think the `asJava`/`asScala` proliferation is too bad - I like having them inline in the code because then when I'm reading the code it's easy to see that a `Map` is not the same as a `Map`